composer.json

```diff
@@ -41,17 +41,13 @@
             },
```

Comment:
+1 for making this documentation only. Using 8.4.x has a certain risk we should not put directly into people's face.